checks/branch_protection_test.go

```diff
@@ -431,9 +431,7 @@ func TestReleaseAndDevBranchProtected(t *testing.T) {
 				RepoClient: mockRepoClient,
 			}
 			r := BranchProtection(&req)
-			if !scut.ValidateTestReturn(t, tt.name, &tt.expected, &r, &dl) {
-				t.Fail()
-			}
+			scut.ValidateTestReturn(t, tt.name, &tt.expected, &r, &dl)
```

Comment:
i'm going to hold this for a bit since there are a few other PRs in flight. I'd rather rebase this one, than have the other PR authors need to do it to theirs.